viewer/apiStats.js

```diff
@@ -17,6 +17,40 @@ const internals = require('./internals');
 const ViewerUtils = require('./viewerUtils');
 
 class StatsAPIs {
+  // --------------------------------------------------------------------------
```

Comment:
@awick can you take a look at the changes in this file? I just want a second pair of eyes on this implementation.